src/components/Totals.js

```diff
@@ -2,47 +2,47 @@ import { Component } from 'preact';
 import { html } from 'htm/preact';
 
 export class Totals extends Component {
+  constructor() {
+    super();
+    this.items = [];
```

Comment:
this is redundant right?

Reply:
Later on the items will be async. But it should be `setState` exactly :)

Reply:
see https://github.com/Code-for-All/lockdown/pull/24/files#r396159844